redash/models.py

```diff
@@ -715,6 +715,32 @@ def recent(cls, groups, user_id=None, limit=20):
 
         return query
 
+    @classmethod
+    def fork(cls, id, user, org):
```

Comment:
Anyway the `fork` method shouldn't fetch the query. It will either operate on itself (if it's an instance method) or on a given query (if we keep it as a class method).

What benefit we have from having this as a class method?